torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -3088,9 +3088,18 @@ def install_subgraph_in_output_graph(
         # installed in the output graph and we can just access the subgraph
         # using the saved attr name.
         from torch._higher_order_ops.utils import has_potential_input_alias_or_mutation
+        from torch._prims_common import clone_preserve_strides
+
+        def _maybe_clone_val(node):
+            example_value = node.meta["example_value"]
+            if isinstance(example_value, torch.Tensor):
+                # We need to clone so that when we retrace, we don't get the
+                # same unbacked symbols
+                return clone_preserve_strides(example_value)
+            return example_value
 
         fake_inputs = [
-            node.meta["example_value"]
+            _maybe_clone_val(node)
```

Comment:
This is not completely correct, if the inputs have view relationships then that matters if there are mutations

Reply:
Resolved offline: this is just used for the alias_mutation_check